common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/azureactivedirectory/AzureActiveDirectory.java

```diff
@@ -201,13 +201,13 @@ public static synchronized void performCloudDiscovery()
             // until the app is killed.
             HttpCache.flush();
 
-            Logger.info(TAG + methodName, "Parsing response.");
-            AzureActiveDirectoryInstanceResponse instanceResponse =
+            final AzureActiveDirectoryInstanceResponse instanceResponse =
                     ObjectMapper.deserializeJsonStringToObject(
                             response.getBody(),
                             AzureActiveDirectoryInstanceResponse.class
                     );
-            Logger.info(TAG + methodName, "Discovered ["
+
+            Logger.verbose(TAG + methodName, "Discovered ["
```

Comment:
My only issue with making these verbose is that CP includes verbose logs, so we'd still get these if CP is the active broker. We should probably tackle that issue by asking CP to change their logging level though.

Reply:
My understanding is that CP disables verbose log in PROD build by default, but i'll double check.

Reply:
Confirmed with CP. They now have INFO as the default.